### DISCUSSION TITLE
DIfferent Function Region for staging and production environment

### DISCUSSION THREAD

**Discussion by tungrix** [vercel/community#3849] [Help]
### Summary

I had my staging and production deployment in the same project by using different GitHub branches, they require different function regions (my staging region is different than the production region).

Is it possible to have a different Function Region for the staging and production deployment? Many thanks.

### Steps to Reproduce

Project -> Project Settings -> Function Region

**Comment by rychlis** [comment]
Also would be very useful for me. I tried changing the vercel.json (`regions` in the JSON) based on $VERCEL_ENV in build step but seems that its ignored
